ts/Accessibility/Components/SeriesComponent/SeriesKeyboardNavigation.ts

```diff
@@ -421,9 +421,19 @@ class SeriesKeyboardNavigation {
                     ): number {
                         const point = chart.highlightedPoint;
                         if (point) {
-                            (event as any).point = point;
+                            const { plotLeft, plotTop } = this.chart,
+                                { plotX = 0, plotY = 0 } = point;
+
+                            event = {
+                                ...(event as any),
+                                chartX: plotLeft + plotX,
+                                chartY: plotTop + plotY,
+                                point: point,
+                                target: point.graphic?.element
+                            };
```

Comment:
What do you think about creating a new file `KeyboardEvent.d.ts` next to `PointerEvent.d.ts`, which will export the extended interface of `KeyboardEvent` (just like it is now for `PointerEvent`), then we can import this type into `SeriesKeyboardNavigation.ts` and the types will be correct, without any 'any's. That would also mean, that the type of event argument of `firePointEvent` would need to change (extended `KeyboardEvent` type).

TL;DR; Create a new extended interface `KeyboardEvent` same as the existing `PointerEvent`.

Reply:
This would make things consistent. Good idea! 👍

Reply:
Sounds good!